app/javascript/lib/vue-components/modules/BeesWarmChart.vue

```diff
@@ -294,8 +303,9 @@ export default {
         d.radius = radiusScale(d[this.radiusProperty])
       });
 
-      let filterData = data.filter(
-        ({ final_amount_no_taxes }) => final_amount_no_taxes !== 0);
+      let filterData = data.filter(({ final_amount_no_taxes }) => final_amount_no_taxes !== 0);
+
+      filterData = filterData.sort((a, b) => a.contract_type.localeCompare(b.contract_type))
```

Comment:
Why do you split this block?

Reply:
And same stuff with fallbacking contract_type

Reply:
Done